test/fixtures/lightscript/if-expression/inner-if-stmt-return/options.json

```diff
@@ -0,0 +1,3 @@
+{
+  "throws": "'return' outside of function (4:6)"
+}
```

Comment:
Yeah, agreed. Wasn't sure how best to do that. I guess a try/catch with rethrow would be easiest but that's a little ugly. Otherwise just add `inExpression` to state, which is more correct.